### DISCUSSION TITLE
Updates on website commissions #1

### DISCUSSION THREAD

**Discussion by JamesTashi** [JamesTashi/JamesTashireposite#5] [General]
Rejected 1.
We have just received no explicit reason only to follow the rules, also I have just 1 account. Hmm....
Hopefully round 2 goes good!

![image](https://user-images.githubusercontent.com/65320859/101580933-66a30b80-398f-11eb-94f3-3b1ad12a0e93.png)

**Comment by BlakeMen** [comment]
F.
Hope it gets approved sooner or later!
